packages/sveltekit/src/vite/sentryVitePlugins.ts

```diff
@@ -17,16 +19,32 @@ type SourceMapsUploadOptions = {
   sourceMapsUploadOptions?: Partial<SentryVitePluginOptions>;
 };
 
+type AutoInstrumentOptions = {
+  /**
+   * The Sentry plugin will automatically instrument certain parts of your SvelteKit application at build time.
+   * Set this option to `false` to disable this behavior or what is instrumentated by passing an object.
+   *
+   * Auto instrumentation includes:
+   * - Universal `load` functions in `+page.(js|ts)` files
+   * - Server-only `load` functions in `+page.server.(js|ts)` files
+   *
+   * @default true (meaning, the plugin will instrument all of the above)
+   */
+  autoInstrument?: boolean | AutoInstrumentSelection;
+};
+
 export type SentrySvelteKitPluginOptions = {
   /**
    * If this flag is `true`, the Sentry plugins will log some useful debug information.
-   * Defaults to `false`.
+   * @default false.
    */
   debug?: boolean;
-} & SourceMapsUploadOptions;
+} & SourceMapsUploadOptions &
+  AutoInstrumentOptions;
 
 const DEFAULT_PLUGIN_OPTIONS: SentrySvelteKitPluginOptions = {
   autoUploadSourceMaps: true,
+  autoInstrument: true,
```

Comment:
Doesn't have to be now but we should think about adding an escape hatch for not auto instrumenting particular routes. I've seen users in nexts that found this useful (eg some people didn't want to instrument their auth routes).

Reply:
Yes, good point. As discussed, I'll do this in a follow-up PR but I agree that it's also a great way of giving people a quick way to opt out of auto instrumenting in case they get an error.